names/names.py

```diff
@@ -11,10 +11,17 @@
 f.close()
 
 duplicates = []
-for name_1 in names_1:
-    for name_2 in names_2:
-        if name_1 == name_2:
-            duplicates.append(name_1)
+# for name_1 in names_1:
+#     for name_2 in names_2:
+#         if name_1 == name_2:
+#             duplicates.append(name_1)
+
+# duplicates.append(str(set(names_1) & set(names_2)))
+
+one = names_1
+two = names_2
+
+duplicates.append(str(set(one) & set(two)))
 
```

Comment:
Nice job utilizing sets for O(1) lookup.  Great use of the intersection operator on these sets to find the duplicates between them.  And I never realized you could .append  a set onto a list, but that great!